ingestion-sink/pom.xml

```diff
@@ -172,6 +172,26 @@
                     <skip>false</skip>
                 </configuration>
             </plugin>
+            <!-- Create an uber jar for standalone usage. https://stackoverflow.com/a/16222971 -->
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-assembly-plugin</artifactId>
+                <version>3.3.0</version>
+                <configuration>
+                    <descriptorRefs>
+                        <descriptorRef>jar-with-dependencies</descriptorRef>
+                    </descriptorRefs>
+                </configuration>
+                <executions>
+                    <execution>
+                        <id>assemble-all</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>single</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
```

Comment:
Uber jars break grpc calls, per [#1283 comment](https://github.com/mozilla/gcp-ingestion/pull/1283/files#r422409954):

> Dependencies have been moved to jars included via this classpath prefix to fix an issue where grpc is breaking because a `jar-with-dependencies` isn't able to include `META-INF/services` from every dependency.